quickstart/src/main/java/com/microsoft/azure/kusto/quickstart/KustoSampleApp.java

```diff
@@ -0,0 +1,395 @@
+            return name;
+        }
+    }
+
+    public static void main(String[] args) throws InterruptedException {
+        System.out.println("Kusto sample app is starting...");
+
+        loadConfigs();
+
+        if (AUTHENTICATION_MODE.equals("UserPrompt")) {
+            waitForUserToProceed("You will be prompted *twice* for credentials during this script. Please return to the console after authenticating.");
+        }
+
+        ConnectionStringBuilder kustoCsb = generateConnectionString(kustoUrl, AUTHENTICATION_MODE);
+        ConnectionStringBuilder ingestCsb = generateConnectionString(ingestUrl, AUTHENTICATION_MODE);
+        Client kustoClient = null;
+        try (IngestClient ingestClient = IngestClientFactory.createClient(ingestCsb)) {
+            // Tip: Avoid creating a new Kusto Client for each use. Instead, create the clients once and reuse them.
+            kustoClient = new ClientImpl(kustoCsb);
+
+            if (!useExistingTable)
+            {
+                waitForUserToProceed(String.format("Create table '%s.%s' if it doesn't exist", databaseName, tableName));
+                // Tip: This is generally a one-time configuration
+                // Learn More: For more information about creating tables, see: https://docs.microsoft.com/azure/data-explorer/one-click-table
+                String command = String.format(".create table %s %s", tableName, tableSchema);
+                if (!execute(kustoClient, databaseName, command, ExecutionType.CONTROL_COMMAND)) {
+                    die("Failed to create table or validate it exists");
+                }
+            }
+
+            /*
+                Learn More:
+                Kusto batches data for ingestion efficiency. The default batching policy ingests data when one of the following conditions are met:
+                 1) More than 1,000 files were queued for ingestion for the same table by the same user
+                 2) More than 1GB of data was queued for ingestion for the same table by the same user
+                 3) More than 5 minutes have passed since the first file was queued for ingestion for the same table by the same user
+                For more information about customizing the ingestion batching policy, see: https://docs.microsoft.com/azure/data-explorer/kusto/management/batchingpolicy
+
+                Tip 1: Though most users should be fine with the defaults, to speed up ingestion, such as during development and
+                 in this sample app, we opt to modify the default ingestion policy to ingest data after 10 seconds have passed.
+                Tip 2: This is generally a one-time configuration.
+                Tip 3: You can also skip the batching for some files using the Flush-Immediately property, though this option should be used with care as it is inefficient.
+             */
+            waitForUserToProceed(String.format("Alter the batching policy for '%s.%s'", databaseName, tableName));

```

Comment:
In retrospect I think this should also fall under the !useExsitingTable condition

Reply:
Without also requiring that `alterTable` is true?

Reply:
See other comment - comment it out, add comment, and move within `createNewTable()`